manifests/cinder-csi-plugin/csi-snapshotter-cinderplugin.yaml

```diff
@@ -0,0 +1,72 @@
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-snapshotter-cinder
+  labels:
+    app: csi-snapshotter-cinder
+spec:
+  selector:
+    app: csi-snapshotter-cinder
+  ports:
+    - name: dummy
+      port: 12345
+
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-snapshotter-cinder
+spec:
+  serviceName: "csi-snapshotter-cinder"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-snapshotter-cinder
+  template:
+    metadata:
+      labels:
+        app: csi-snapshotter-cinder
+    spec:
+      serviceAccount: csi-snapshotter
+      containers:
+        - name: csi-snapshotter
+          image: quay.io/k8scsi/csi-snapshotter:v0.4.0
```

Comment:
shouldn't we use v1.0.1 release here which supports csi spec 1.0?

Reply:
yes, it was 1.0.1, dont know what happened while rebasing :), will check and update